detection-rules/impersonation_org_sld_in_sender_local_fts.yml

```
@@ -0,0 +1,25 @@
+name: "Impersonation: Organizational SLD found in Sender Local FTS"
+type: "query"
+description: |
+  The Sender local contains an organizational SLD,
+  and the sender's domain is not in org domains and first time sender.
+source: |
+  type.inbound
+  and ( 
+    any($org_slds, strings.contains(sender.email.local_part, .))
+      and sender.email.domain.root_domain not in $org_domains
+  )
```

a little more readable might be:

```suggestion
  and any($org_slds, strings.contains(sender.email.local_part, .))
  and sender.email.domain.root_domain not in $org_domains
```